src/game/client/components/chat.cpp

```diff
@@ -385,6 +385,14 @@ bool CChat::OnInput(IInput::CEvent Event)
 				// add part before the name
 				str_truncate(aBuf, sizeof(aBuf), m_Input.GetString(), m_PlaceholderOffset);
 
+				// quote the name
+				char aQuoted[MAX_NAME_LENGTH+2];
+				if(IsTypingCommand() && *str_skip_to_whitespace_const(pCompletionString) != '\0')
```

Comment:
```suggestion
				if(IsTypingCommand() && str_find(pCompletionString, " "))
```

Would look better

Reply:
Yes but can names include tabs? I wanted to go save.